video/out/vo_dmabuf_wayland.c

```diff
@@ -677,7 +677,18 @@ static int reconfig(struct vo *vo, struct mp_image *img)
     if (!vo_wayland_reconfig(vo))
         return VO_ERROR;
 
-    wl_surface_set_buffer_transform(vo->wl->video_surface, img->params.rotate / 90);
+    // mpv rotates clockwise but the wayland spec has counter-clockwise rotations
+    // swap 1 and 3 to match mpv's direction
+    int transform = img->params.rotate / 90;
+    switch (transform) {
+    case 1:
+        transform = 3;
+        break;
+    case 3:
+        transform = 1;
+        break;
+    }
```

Comment:
```suggestion
    int transform = (360 - img->params.rotate) % 360 / 90;
```

Reply:
Of course there was a smarter way to do this.